Readme.md

```diff
@@ -192,7 +192,7 @@ expect(fn).not.toBeCalled() // Function was *not* called
 expect(fn).toHaveBeenCalledTimes(1) // Function was called only once
 expect(fn).toBeCalledWith(arg1, arg2) // Any of calls was with these arguments
 expect(fn).toHaveBeenLastCalledWith(arg1, arg2) // Last call was with these arguments
-expect(fn).toHaveBeenNthCalledWith(args) // Nth call was with these arguments (Jest 23+)
+expect(fn).toHaveBeenNthCalledWith(nthCall, args) // Nth call was with these arguments (Jest 23+)
```

Comment:
Maybe we should make the name clearer. I think `nthCall` is quite confusing without examples.

```suggestion
expect(fn).toHaveBeenNthCalledWith(callNumber, args) // Nth call was with these arguments (Jest 23+)
```

Reply:
Agreed!